recipe/dashboard/api/usersGet.go

```diff
@@ -64,7 +66,24 @@ func UsersGet(apiImplementation dashboardmodels.APIInterface, options dashboardm
 
 	var usersResponse supertokens.UserPaginationResult
 
-	if timeJoinedOrder == "ASC" {
+	u, err := url.Parse(req.URL.String())
+	if err != nil {
+		return UsersGetResponse{}, err
+	}
+
+	queryParams, err := url.ParseQuery(u.RawQuery)
+	if err != nil {
+		return UsersGetResponse{}, err
+	}
+
+	queryParamsObject := map[string]string{}
+	for i, s := range queryParams {
+		queryParamsObject[i] = strings.Join(s, ";")
+	}
+
+	if len(queryParamsObject) != 0 {
+		usersResponse, err = supertokens.GetUsersSearchParams(timeJoinedOrder, paginationTokenPtr, &limit, nil, queryParamsObject)
```

Comment:
Use the internal function here, this function does not need to be exposed to the user